sdk/eventgrid/eventgrid/test/public/utils/recordedClient.ts

```diff
@@ -2,18 +2,12 @@
 // Licensed under the MIT license.
 
 import { Context } from "mocha";
-import * as dotenv from "dotenv";
 
 import { env, Recorder, record, RecorderEnvironmentSetup } from "@azure-tools/test-recorder";
-import { isNode } from "./testUtils";
 
 import { EventGridPublisherClient, InputSchema } from "../../../src";
 import { KeyCredential } from "@azure/core-auth";
 
-if (isNode) {
-  dotenv.config();
-}
```

Comment:
NIT: I wonder if we need to care about future-proofing tests to accommodate the scenario where a new test suite is added that does not use the recorder but still needs to load environment variables. No need to address this in anyway in this PR though.